Find a functionally equivalent code:android.animation.ObjectAnimator.getTarget:COMMENT
Method Modifier: public      
Comment:/**
 * The target object whose property will be animated by this animation
 *
 * @return The object being animated
 */

@@ -1,3 +1,3 @@
 {
-    return mTarget;
+    return mTarget == null ? null : mTarget.get();
 }

